pkg/dump/dump.go

```diff
@@ -255,18 +252,25 @@ func getProxyConfiguration(ctx context.Context, group *errgroup.Group,
 		return nil
 	})
 
-	if config.IsFilterChainsSupported {
-		group.Go(func() error {
-			filterChains, err := GetAllFilterChains(ctx, client, config.SelectorTags)
-			if err != nil {
-				return fmt.Errorf("filter chains: %w", err)
-			}
-			state.FilterChains = filterChains
-			return nil
-		})
-	} else {
+	group.Go(func() error {
 		state.FilterChains = make([]*kong.FilterChain, 0)
```

Comment:
We don't really need to set this here and allocate the slice, right?

Line 271 will do I guess.

Reply:
You're right--we don't strictly _need_ to allocate here. I thought it better to have the "set a default value" code at the top rather than hiding it 3 levels deep in the "we got an error, but it's okay" branch, but I have no objections to moving it.